freetextresponse/__init__.py

```diff
@@ -0,0 +1,6 @@
+'''
+This is an XBlock that accepts a free-text response from students.
+Instructors can specify a list of phrases, of which one must be
+present in order for the student to receive credit.
+'''
+from .freetextresponse import Freetextresponse
```

Comment:
I wonder if we need this or if it's mostly cargo cult at this point..?

Correspondingly, we may need to update the `entry_points` in `setup.py` (if we remove this line here:
```python
entry_points={
    'xblock.v1': [
        'freetextresponse = freetextresponse.freetextresponse:Freetextresponse',
    ],
},
```

Mind giving it a try?